nifi-extension-bundles/nifi-confluent-platform-bundle/nifi-confluent-schema-registry-service/pom.xml

```diff
@@ -77,5 +77,29 @@
             <artifactId>nifi-web-utils</artifactId>
             <version>2.6.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-confluent-protobuf-message-name-resolver</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <!-- Test dependencies -->
+
+        <dependency>
+            <groupId>com.github.tomakehurst</groupId>
+            <artifactId>wiremock-jre8-standalone</artifactId>
+            <version>2.35.0</version>
```

Comment:
This is an old version of Wiremock, and other HTTP client classes in other modules use OkHttp MockWebServer, so this should be changed to follow that pattern.